doc/taskwiki.txt

```diff
@@ -382,8 +382,28 @@ viewport definition:
 
 ~   == Differently ordered home tasks | project:Home $T ==
 
+------------------
+5.2.4 Limiting number of tasks displayed
+
+By default, taskwiki will present the full list of tasks that match the
+viewport definition.
+
+You can limit the number of tasks listed by using the limit filter:
+
+~   == List only the 5 first tasks | project:Home $T limit:5 ==
```

Comment:
I think you only wanted to mention `$T` in the second example.

Reply:
Yes, you are right!